java/code/src/com/redhat/rhn/taskomatic/task/TaskHelper.java

```diff
@@ -80,24 +73,17 @@ public static void sendErrorEmail(String messageBody) {
      * @param messageBody to send.
      */
     public static void sendTaskoEmail(Integer orgId, String messageBody) {
-        Config c = Config.get();
         LocalizationService ls = LocalizationService.getInstance();
         String[] recipients = MailHelper.getAdminRecipientsFromConfig();
         if (orgId != null) {
             List<String> emails = getActiveOrgAdminEmails(orgId);
             recipients = !emails.isEmpty() ?
-                    emails.toArray(new String[emails.size()]) :
+                    emails.toArray(new String[0]) :
```

Comment:
I don't get why this change?

Reply:
Was a sonarcloud suggestion. toArray takes care that all the elements are returned. The parameter should only give the type. It may make difference when you want to return an array which is bigger than the number of elements, but same size should also work this way.